tests/src/init.rs

```diff
@@ -27,7 +27,12 @@ pub fn initialize_instance() -> Instance {
     } else {
         Backends::all()
     };
-    let dx12_shader_compiler = wgpu::util::dx12_shader_compiler_from_env().unwrap_or_default();
+    // TODO: should env override force_fxc?
```

Comment:
No - tests need to be able to force demote it.

Reply:
Thanks! Removed TODO comment and added a small note about this.